web-server/v0.4/src/components/PageHeader/index.js

```diff
@@ -1,7 +1,7 @@
 import React, { PureComponent, createElement } from 'react';
 import PropTypes from 'prop-types';
 import pathToRegexp from 'path-to-regexp';
-import { Breadcrumb, Tabs } from 'antd';
+import { Breadcrumb, Tabs, Tag, Col } from 'antd';
```

Comment:
Are `Tag` and `Col` predefined to properly display title-case?

Reply:
These are predefined components of Ant-Design. However we don't need Col anymore.